symfit/core/fit.py

```diff
@@ -1499,6 +1499,13 @@ def _init_minimizer(self, minimizer, **minimizer_options):
             # py function version of the analytical jacobian.
             if hasattr(self.model, 'numerical_jacobian') and hasattr(self.objective, 'eval_jacobian'):
                 minimizer_options['jacobian'] = self.objective.eval_jacobian
+        if issubclass(minimizer, HessianMinimizer):
+            # If an analytical version of the Hessian exists we should use
+            # that, otherwise we let the minimizer estimate it itself.
+            # Hence the check of numerical_hessian, as this is the
+            # py function version of the analytical jacobian.
+            if hasattr(self.model, 'numerical_hessian') and hasattr(self.objective, 'eval_hessian'):
+                minimizer_options['hessian'] = self.objective.eval_hessian
```

Comment:
This contradicts the comment. And is numerical_hessian still a thing or did it get trimmed recently?

Reply:
In #201 the check for `model.numerical_xxx` will be replaced by `model.eval_xxx`. But I think we should merge this PR first, and then I will rebase #201. In #201 I have removed `numerical_jacobian` and `numerical_hessian` in favor of giving Hessians and Jacobians there own model, so this well definitely make some tests fail so I cannot forget to update this line. So I would say leave it like this for this PR.

Reply:
> So I would say leave it like this for this PR.

Best possible answer :D